README.md

```diff
@@ -15,18 +15,27 @@ This project assumes that you will run this on [a Raspberry Pi][pi] with a CSI-p
 
 #### Installing [`FFmpeg`][ffmpeg]
 
-FFmpeg is used to convert the sequence of photos captured by the camera into a video suitable for uploading to Twitter. FFmpeg must be compiled with x264 support. On Raspberry Pi I found [these instructions][ffmpeg-pi] to be helpful. If you are running this on a Mac, `brew install ffpmeg` should be sufficient.
+FFmpeg is used to convert the sequence of photos captured by the camera into a video suitable for uploading to Twitter. FFmpeg must be compiled with x264 support. On a Raspberry Pi running Raspbian, simply `sudo apt install ffmpeg`. If you are running this on a Mac, `brew install ffpmeg` should be sufficient.
 
 [ffmpeg]: http://ffmpeg.org
-[ffmpeg-pi]: http://www.jeffreythompson.org/blog/2014/11/13/installing-ffmpeg-for-raspberry-pi/
 
 #### Installing `golden-hour`
 
 1. Check out this repo to a convenient location on your Pi.
-2. Run `pip install -r requirements` (I recommend doing this in a virtualenv)
+2. Run `pip install .`
```

Comment:
Once you get this published, these steps should be changed right?

Reply:
yeah, though I'm thinking of these changes as unblocking #1 rather than actually doing it